a-braham/Questioner
# #163347065 Modified the meetups page and responsiveness

#### What does this PR do?
 * The pull request modifies meetups.html page

#### Description of Task to be completed?
  **Have the following features complete**
  * Page should be responsive
  * Should not have any inline css
  * User should be logged in for "Ask question" button to be visible

#### How should this be manually tested?
  * Clone the repository
  * Open the /meetups.html file with chrome
  * Test for responsiveness, check the html tags and css for errors
 
#### What are the relevant pivotal tracker stories?
  #163347065

#### Screenshots
![screenshot from 2019-01-20 11-25-23](https://user-images.githubusercontent.com/45788436/51436881-2c9ac480-1ca6-11e9-9dd1-566c2eccca36.png)
![screenshot from 2019-01-20 11-25-51](https://user-images.githubusercontent.com/45788436/51436882-2c9ac480-1ca6-11e9-9884-3a7c34e83c8c.png)
